### DISCUSSION TITLE
Is it possible to rename WasmScripts and WasmCSS folders in the Web Assembly Project?

### DISCUSSION THREAD

**Discussion by keremo** [unoplatform/uno#11933] [Q&A]
I tried renaming those folders in Visual Studio and the project can be built without issues. However, when I try and run the web assembly project nothing is loaded with several errors in the browser's console.

Where are those paths coming from? Is it possible rename them or to change the project configuration to use different paths for AppManifest.js and Fonts.css?

Thanks in advance for any response.

**Comment by jeromelaban** [comment]
At this time it is not, but we're tracking this with https://github.com/unoplatform/Uno.Wasm.Bootstrap/issues/291.
